packages/cli/src/cmds/beacon/options.ts

```diff
@@ -140,6 +141,11 @@ export const beaconExtraOptions: CliCommandOptions<BeaconExtraArgs> = {
     description: "Attach the beacon node to `globalThis`. Useful to inspect a running beacon node.",
     type: "boolean",
   },
+
+  disableLightClientServer: {
+    description: "Disable lightclient server.",
```

Comment:
for everything user facing in text we might wanna use "light client"

```suggestion
    description: "Disable light client server.",
```